Dockerfile

```diff
@@ -1,5 +1,6 @@
-FROM golang:1.14 AS build
+FROM registry.redhat.io/ubi8/ubi AS build
 WORKDIR /go/src/app
+RUN yum -y install golang make
```

Comment:
I would also do `yum clean all` but since it is intermediate image just to build the binary and then final image is built using the scratch so may be just ignore.